contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol

```diff
@@ -636,6 +639,14 @@ contract OracleReportSanityChecker is AccessControlEnumerable {
             revert TooHighSimulatedShareRate(_simulatedShareRate, actualShareRate);
         }
 
+        if (_simulatedShareRate >= _provedMinShareRate) {
```

Comment:
I wouldn't go so far, but let's discuss

Reply:
Spent the night with this and have the same feeling: let's exclude `_provedMinShareRate` from this sanity check at all.